### ISSUE TITLE
Anzahl der Treffer in der Trefferliste

### ISSUE THREAD

**Issue by jenniferward** [rism-international/opac-report#102]
In der Trefferliste: Aktuell werden 10 Titel als Standardeinstellung angezeigt. Aus der Nutzerstudie wissen wir, dass kaum jemand nach der ersten Ergebnissseite blättert, obwohl die gesuchte Quelle möglicherweise dort zu finden wäre.

Die Titelanzahl (50) soll voreingestellt sein und zumindest pro Sitzung beibehalten werden.

**Comment by BernLutz** [comment]
In VuFind werden in der Standardeinstellung nun 20 Titel angezeigt. Wäre das eine praktikable Lösung oder sehen Sie hier noch Handlungsbedarf  @jenniferward ?

**Comment by BaMikusi** [comment]
Aus meiner Sicht wäre 50 als Grundeinstellung weiterhin besser. 20 mag auf dem Bildschirm netter aussehen, aber 50 erhöht die Chancen, dass das Gesuchte tatsächlich gefunden wird.

**Comment by jenniferward** [comment]
Ja, 50 sind tatsächlich besser. Ich wechsele immer grundsätzlich auf das Maximum, um mit strg+F effektiver zu suchen.
